[2021-12-20 11:21:56,831] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_wexford.scrap_data scheduled__2021-12-19T11:05:00+00:00 [queued]>
[2021-12-20 11:21:57,252] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_wexford.scrap_data scheduled__2021-12-19T11:05:00+00:00 [queued]>
[2021-12-20 11:21:57,258] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-20 11:21:57,264] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-20 11:21:57,269] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-20 11:21:57,661] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-19 11:05:00+00:00
[2021-12-20 11:21:57,738] {standard_task_runner.py:52} INFO - Started process 5879 to run task
[2021-12-20 11:21:57,862] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_wexford', 'scrap_data', 'scheduled__2021-12-19T11:05:00+00:00', '--job-id', '3440', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_wexford.py', '--cfg-path', '/tmp/tmptk07mpy0', '--error-file', '/tmp/tmp8rs92ud4']
[2021-12-20 11:21:57,936] {standard_task_runner.py:77} INFO - Job 3440: Subtask scrap_data
[2021-12-20 11:22:02,776] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_wexford.scrap_data scheduled__2021-12-19T11:05:00+00:00 [running]> on host a35fe600e898
[2021-12-20 11:22:07,696] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_wexford
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-19T11:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-19T11:05:00+00:00
[2021-12-20 11:22:07,857] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-20 11:22:07,871] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py wexford
[2021-12-20 11:23:10,806] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-20 11:23:12,967] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-20 11:23:12,975] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-20 11:23:13,001] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-20 11:23:13,004] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-20 11:23:13,045] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-20 11:23:14,162] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-20 11:23:14,175] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-20 11:23:14,177] {spark_submit.py:514} INFO - deployMode              null
[2021-12-20 11:23:14,179] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-20 11:23:14,181] {spark_submit.py:514} INFO - executorCores           null
[2021-12-20 11:23:14,184] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-20 11:23:14,186] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-20 11:23:14,188] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-20 11:23:14,190] {spark_submit.py:514} INFO - driverCores             null
[2021-12-20 11:23:14,193] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-20 11:23:14,195] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-20 11:23:14,199] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-20 11:23:14,202] {spark_submit.py:514} INFO - supervise               false
[2021-12-20 11:23:14,203] {spark_submit.py:514} INFO - queue                   null
[2021-12-20 11:23:14,206] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-20 11:23:14,207] {spark_submit.py:514} INFO - files                   null
[2021-12-20 11:23:14,208] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-20 11:23:14,210] {spark_submit.py:514} INFO - archives                null
[2021-12-20 11:23:14,211] {spark_submit.py:514} INFO - mainClass               null
[2021-12-20 11:23:14,218] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-20 11:23:14,221] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-20 11:23:14,223] {spark_submit.py:514} INFO - childArgs               [wexford]
[2021-12-20 11:23:14,224] {spark_submit.py:514} INFO - jars                    null
[2021-12-20 11:23:14,226] {spark_submit.py:514} INFO - packages                null
[2021-12-20 11:23:14,227] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-20 11:23:14,229] {spark_submit.py:514} INFO - repositories            null
[2021-12-20 11:23:14,231] {spark_submit.py:514} INFO - verbose                 true
[2021-12-20 11:23:14,232] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:14,235] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-20 11:23:14,238] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-20 11:23:14,240] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-20 11:23:14,242] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:14,252] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:30,270] {spark_submit.py:514} INFO - Main class:
[2021-12-20 11:23:30,276] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-20 11:23:30,281] {spark_submit.py:514} INFO - Arguments:
[2021-12-20 11:23:30,283] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-20 11:23:30,285] {spark_submit.py:514} INFO - null
[2021-12-20 11:23:30,286] {spark_submit.py:514} INFO - wexford
[2021-12-20 11:23:30,287] {spark_submit.py:514} INFO - --verbose
[2021-12-20 11:23:30,307] {spark_submit.py:514} INFO - Spark config:
[2021-12-20 11:23:30,312] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-20 11:23:30,315] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-20 11:23:30,316] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-20 11:23:30,319] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-20 11:23:30,320] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-20 11:23:30,321] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:30,322] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:30,324] {spark_submit.py:514} INFO - 
[2021-12-20 11:23:43,016] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-20 11:23:43,200] {spark_submit.py:514} INFO - 21/12/20 11:23:43 INFO SparkContext: Running Spark version 3.2.0
[2021-12-20 11:23:44,421] {spark_submit.py:514} INFO - 21/12/20 11:23:44 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-20 11:23:46,038] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO ResourceUtils: ==============================================================
[2021-12-20 11:23:46,041] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-20 11:23:46,042] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO ResourceUtils: ==============================================================
[2021-12-20 11:23:46,048] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-20 11:23:46,236] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-20 11:23:46,364] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO ResourceProfile: Limiting resource is cpu
[2021-12-20 11:23:46,367] {spark_submit.py:514} INFO - 21/12/20 11:23:46 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-20 11:23:47,164] {spark_submit.py:514} INFO - 21/12/20 11:23:47 INFO SecurityManager: Changing view acls to: ***
[2021-12-20 11:23:47,183] {spark_submit.py:514} INFO - 21/12/20 11:23:47 INFO SecurityManager: Changing modify acls to: ***
[2021-12-20 11:23:47,188] {spark_submit.py:514} INFO - 21/12/20 11:23:47 INFO SecurityManager: Changing view acls groups to:
[2021-12-20 11:23:47,190] {spark_submit.py:514} INFO - 21/12/20 11:23:47 INFO SecurityManager: Changing modify acls groups to:
[2021-12-20 11:23:47,192] {spark_submit.py:514} INFO - 21/12/20 11:23:47 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-20 11:23:50,065] {spark_submit.py:514} INFO - 21/12/20 11:23:50 INFO Utils: Successfully started service 'sparkDriver' on port 37249.
[2021-12-20 11:23:50,462] {spark_submit.py:514} INFO - 21/12/20 11:23:50 INFO SparkEnv: Registering MapOutputTracker
[2021-12-20 11:23:51,240] {spark_submit.py:514} INFO - 21/12/20 11:23:51 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-20 11:23:51,701] {spark_submit.py:514} INFO - 21/12/20 11:23:51 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-20 11:23:51,876] {spark_submit.py:514} INFO - 21/12/20 11:23:51 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-20 11:23:51,877] {spark_submit.py:514} INFO - 21/12/20 11:23:51 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-20 11:23:52,064] {spark_submit.py:514} INFO - 21/12/20 11:23:52 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-9a77b37b-789e-472d-bd15-7a4f278334fc
[2021-12-20 11:23:52,571] {spark_submit.py:514} INFO - 21/12/20 11:23:52 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-20 11:23:52,896] {spark_submit.py:514} INFO - 21/12/20 11:23:52 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-20 11:23:56,217] {spark_submit.py:514} INFO - 21/12/20 11:23:56 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-20 11:23:56,244] {spark_submit.py:514} INFO - 21/12/20 11:23:56 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-20 11:23:56,299] {spark_submit.py:514} INFO - 21/12/20 11:23:56 INFO Utils: Successfully started service 'SparkUI' on port 4042.
[2021-12-20 11:23:56,967] {spark_submit.py:514} INFO - 21/12/20 11:23:56 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://a35fe600e898:4042
[2021-12-20 11:23:59,539] {spark_submit.py:514} INFO - 21/12/20 11:23:59 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-20 11:24:00,353] {spark_submit.py:514} INFO - 21/12/20 11:24:00 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-20 11:24:00,354] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-20 11:24:00,356] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-20 11:24:00,359] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-20 11:24:00,361] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-20 11:24:00,364] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-20 11:24:00,366] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-20 11:24:00,368] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-20 11:24:00,372] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-20 11:24:00,373] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-20 11:24:00,375] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-20 11:24:00,376] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:24:00,377] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-20 11:24:00,379] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-20 11:24:00,381] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-20 11:24:00,383] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-20 11:24:00,384] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-20 11:24:00,385] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-20 11:24:00,386] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-20 11:24:00,388] {spark_submit.py:514} INFO - ... 4 more
[2021-12-20 11:24:00,389] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-20 11:24:00,391] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-20 11:24:00,392] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-20 11:24:00,394] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-20 11:24:00,395] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-20 11:24:00,396] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-20 11:24:00,400] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-20 11:24:00,402] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-20 11:24:00,403] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-20 11:24:00,405] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-20 11:24:00,406] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-20 11:24:00,408] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-20 11:24:00,409] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-20 11:24:00,411] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-20 11:24:00,413] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-20 11:24:00,414] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-20 11:24:00,416] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-20 11:24:00,418] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-20 11:24:00,421] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-20 11:24:00,425] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-20 11:24:00,426] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-20 11:24:00,427] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-20 11:24:00,429] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-20 11:24:00,431] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-20 11:24:00,432] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-20 11:24:00,433] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-20 11:24:00,435] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-20 11:24:00,437] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-20 11:24:00,439] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-20 11:24:00,440] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-20 11:24:00,442] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-20 11:24:00,443] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-20 11:24:00,445] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-20 11:24:00,446] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-20 11:24:00,448] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-20 11:24:00,450] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-20 11:24:00,457] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-20 11:24:00,459] {spark_submit.py:514} INFO - ... 1 more
[2021-12-20 11:24:19,466] {spark_submit.py:514} INFO - 21/12/20 11:24:19 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-20 11:24:19,491] {spark_submit.py:514} INFO - 21/12/20 11:24:19 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-20 11:24:19,493] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-20 11:24:19,495] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-20 11:24:19,496] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-20 11:24:19,498] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-20 11:24:19,499] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-20 11:24:19,502] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-20 11:24:19,506] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-20 11:24:19,509] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-20 11:24:19,510] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-20 11:24:19,511] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-20 11:24:19,512] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:24:19,513] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-20 11:24:19,514] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-20 11:24:19,515] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-20 11:24:19,516] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-20 11:24:19,517] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-20 11:24:19,518] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-20 11:24:19,519] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-20 11:24:19,520] {spark_submit.py:514} INFO - ... 4 more
[2021-12-20 11:24:19,521] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-20 11:24:19,522] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-20 11:24:19,523] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-20 11:24:19,524] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-20 11:24:19,525] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-20 11:24:19,526] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-20 11:24:19,528] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-20 11:24:19,529] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-20 11:24:19,530] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-20 11:24:19,531] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-20 11:24:19,532] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-20 11:24:19,534] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-20 11:24:19,535] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-20 11:24:19,536] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-20 11:24:19,538] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-20 11:24:19,539] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-20 11:24:19,541] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-20 11:24:19,542] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-20 11:24:19,543] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-20 11:24:19,544] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-20 11:24:19,546] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-20 11:24:19,547] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-20 11:24:19,548] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-20 11:24:19,549] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-20 11:24:19,550] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-20 11:24:19,551] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-20 11:24:19,552] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-20 11:24:19,553] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-20 11:24:19,555] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-20 11:24:19,555] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-20 11:24:19,557] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-20 11:24:19,557] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-20 11:24:19,558] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-20 11:24:19,559] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-20 11:24:19,560] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-20 11:24:19,561] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-20 11:24:19,562] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-20 11:24:19,563] {spark_submit.py:514} INFO - ... 1 more
[2021-12-20 11:24:39,432] {spark_submit.py:514} INFO - 21/12/20 11:24:39 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-20 11:24:39,452] {spark_submit.py:514} INFO - 21/12/20 11:24:39 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-20 11:24:39,454] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-20 11:24:39,455] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-20 11:24:39,456] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-20 11:24:39,457] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-20 11:24:39,458] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-20 11:24:39,459] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-20 11:24:39,460] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-20 11:24:39,461] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-20 11:24:39,462] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-20 11:24:39,462] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-20 11:24:39,464] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:24:39,465] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-20 11:24:39,466] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-20 11:24:39,467] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-20 11:24:39,468] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-20 11:24:39,469] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-20 11:24:39,469] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-20 11:24:39,470] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-20 11:24:39,471] {spark_submit.py:514} INFO - ... 4 more
[2021-12-20 11:24:39,472] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-20 11:24:39,473] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-20 11:24:39,475] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-20 11:24:39,475] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-20 11:24:39,476] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-20 11:24:39,477] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-20 11:24:39,478] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-20 11:24:39,478] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-20 11:24:39,479] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-20 11:24:39,480] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-20 11:24:39,481] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-20 11:24:39,481] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-20 11:24:39,482] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-20 11:24:39,483] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-20 11:24:39,483] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-20 11:24:39,484] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-20 11:24:39,485] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-20 11:24:39,485] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-20 11:24:39,486] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-20 11:24:39,487] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-20 11:24:39,487] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-20 11:24:39,488] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-20 11:24:39,489] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-20 11:24:39,490] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-20 11:24:39,491] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-20 11:24:39,491] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-20 11:24:39,492] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-20 11:24:39,493] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-20 11:24:39,494] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-20 11:24:39,495] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-20 11:24:39,495] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-20 11:24:39,496] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-20 11:24:39,497] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-20 11:24:39,498] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-20 11:24:39,498] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-20 11:24:39,499] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-20 11:24:39,500] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-20 11:24:39,501] {spark_submit.py:514} INFO - ... 1 more
[2021-12-20 11:24:59,448] {spark_submit.py:514} INFO - 21/12/20 11:24:59 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-20 11:24:59,464] {spark_submit.py:514} INFO - 21/12/20 11:24:59 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-20 11:24:59,566] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 34231.
[2021-12-20 11:24:59,581] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO NettyBlockTransferService: Server created on a35fe600e898:34231
[2021-12-20 11:24:59,610] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-20 11:24:59,655] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, a35fe600e898, 34231, None)
[2021-12-20 11:24:59,677] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO SparkUI: Stopped Spark web UI at http://a35fe600e898:4042
[2021-12-20 11:24:59,682] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO BlockManagerMasterEndpoint: Registering block manager a35fe600e898:34231 with 434.4 MiB RAM, BlockManagerId(driver, a35fe600e898, 34231, None)
[2021-12-20 11:24:59,702] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, a35fe600e898, 34231, None)
[2021-12-20 11:24:59,722] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, a35fe600e898, 34231, None)
[2021-12-20 11:24:59,768] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-20 11:24:59,798] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-20 11:24:59,875] {spark_submit.py:514} INFO - 21/12/20 11:24:59 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-20 11:24:59,959] {spark_submit.py:514} INFO - 21/12/20 11:24:59 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-20 11:25:00,284] {spark_submit.py:514} INFO - 21/12/20 11:25:00 INFO MemoryStore: MemoryStore cleared
[2021-12-20 11:25:00,298] {spark_submit.py:514} INFO - 21/12/20 11:25:00 INFO BlockManager: BlockManager stopped
[2021-12-20 11:25:00,402] {spark_submit.py:514} INFO - 21/12/20 11:25:00 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-20 11:25:00,414] {spark_submit.py:514} INFO - 21/12/20 11:25:00 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-20 11:25:00,551] {spark_submit.py:514} INFO - 21/12/20 11:25:00 INFO SparkContext: Successfully stopped SparkContext
[2021-12-20 11:25:00,865] {spark_submit.py:514} INFO - 21/12/20 11:25:00 ERROR SparkContext: Error initializing SparkContext.
[2021-12-20 11:25:00,867] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-20 11:25:00,872] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-20 11:25:00,879] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-20 11:25:00,881] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-20 11:25:00,887] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-20 11:25:00,890] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-20 11:25:00,892] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-20 11:25:00,894] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-20 11:25:00,896] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-20 11:25:00,899] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-20 11:25:00,902] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-20 11:25:00,904] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-20 11:25:00,905] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-20 11:25:00,908] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-20 11:25:00,910] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-20 11:25:00,913] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-20 11:25:00,916] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:25:00,919] {spark_submit.py:514} INFO - 21/12/20 11:25:00 INFO SparkContext: SparkContext already stopped.
[2021-12-20 11:25:00,922] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-20 11:25:00,924] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-20 11:25:00,926] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-20 11:25:00,930] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-20 11:25:00,933] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-20 11:25:00,945] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-20 11:25:00,949] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-20 11:25:00,951] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-20 11:25:00,953] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-20 11:25:00,954] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-20 11:25:00,956] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-20 11:25:00,957] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-20 11:25:00,959] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-20 11:25:00,960] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-20 11:25:00,963] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-20 11:25:00,966] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-20 11:25:00,969] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-20 11:25:00,972] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-20 11:25:00,977] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-20 11:25:00,978] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-20 11:25:00,980] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-20 11:25:00,981] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-20 11:25:00,983] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-20 11:25:00,984] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-20 11:25:00,987] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-20 11:25:00,988] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-20 11:25:00,991] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-20 11:25:00,994] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-20 11:25:01,002] {spark_submit.py:514} INFO - 
[2021-12-20 11:25:01,295] {spark_submit.py:514} INFO - 21/12/20 11:25:01 INFO ShutdownHookManager: Shutdown hook called
[2021-12-20 11:25:01,299] {spark_submit.py:514} INFO - 21/12/20 11:25:01 INFO ShutdownHookManager: Deleting directory /tmp/spark-aae21ca2-fa1e-4c66-a1fe-00d2050775ce
[2021-12-20 11:25:01,305] {spark_submit.py:514} INFO - 21/12/20 11:25:01 INFO ShutdownHookManager: Deleting directory /tmp/spark-af7bd451-3d92-44de-a1ad-7b88decee031
[2021-12-20 11:25:02,551] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py wexford. Error code is: 1.
[2021-12-20 11:25:02,662] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_wexford, task_id=scrap_data, execution_date=20211219T110500, start_date=20211220T112156, end_date=20211220T112502
[2021-12-20 11:25:02,783] {standard_task_runner.py:91} ERROR - Failed to execute job 3440 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py wexford. Error code is: 1.
[2021-12-20 11:25:02,993] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-20 11:25:03,731] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
